### PR TITLE
Bug 1274200: Prompt by default when deleting a non-existent tag

### PR DESCRIPTION
**Pull request by kargakis**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1274200

### CODE REVIEW

**Comment by kargakis** [comment]
@soltysh PTAL

**Comment by soltysh** [comment]
Per IRC discussion, I'd prefer this command to actually fail with non-zero exit code, rather than printing that info to the user.

**Comment by mfojtik** [comment]
@soltysh print the info to stderr and exit 1

File: pkg/cmd/cli/cmd/tag.go [hunk 1]
```diff
@@ -303,7 +303,7 @@ func (o TagOptions) RunTag() error {
 		} else {
 			// The user wants to delete a spec tag.
 			if _, ok := target.Spec.Tags[destTag]; !ok {
-				glog.V(4).Infof("Destination tag %s/%s does not exist", o.destNamespace[i], destNameAndTag)
+				fmt.Printf("Destination tag %s/%s does not exist.\n", o.destNamespace[i], destNameAndTag)
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/tag.go]: Use the cmd's `out`

Comment (kargakis) [inline pkg/cmd/cli/cmd/tag.go]: We still have to return an error so I just return instead of logging.

**Comment by soltysh** [comment]
LGTM [merge]

**Pull request merged**